ui/src/pages/Create.js

```diff
@@ -389,11 +461,12 @@ class Create extends Component {
     );
   }
 
-  renderTextFieldAndInput(formKey, inputName, labelMessage) {
+  renderTextFieldOrAreaAndInput(formKey, inputName, labelMessage, textArea) {
```

Comment:
as discussed offline, make this two separate methods for clarity

Reply:
Done.